src/Test/TestCases.Activities/TerminateWorkflowTests.cs

```diff
@@ -208,7 +208,7 @@ private static void runTerminate()
 
         private class MyTracking : TrackingParticipant
         {
-            protected override void Track(TrackingRecord record, TimeSpan timeout)
+            protected internal override void Track(TrackingRecord record, TimeSpan timeout)
```

Comment:
Build was erroring with following error:
```
'TerminateWorkflowTests.MyTracking.Track(TrackingRecord, TimeSpan)': cannot change access modifiers when overriding 'protected internal' inherited member 'TrackingParticipant.Track(TrackingRecord, TimeSpan)'
```
Adding `internal` was suggested to match the method it was overriding, so the suggestion was applied.

Reply:
The project builds and the CI proves that :) Just revert this unrelated change.